dosomething-dev/fastly-frontend/legacy_redirects_recv.vcl

```diff
@@ -0,0 +1,14 @@
+# Redirect */actnow/* page requests.
+if (req.url ~ "(?i)^\/((us|mx|br)\/)?actnow\/") { error 800 "Not Found"; }
```

Comment:
Instead of three separate error codes, what if we shared `800` for all of these and passed a parameter? I haven't tested this out before, but I think it might work...

```vcl
if (req.url ~ "(?i)^\/((us|mx|br)\/)?actnow\/") { error 800 "/us"; }
```

```
if (obj.status == 800) {
  set obj.http.Location = obj.response;
  set obj.status = 301;
  return (deliver);
}
```

Reply:
🤔 yeah, potentially. Does having the separate codes help in terms of knowing what's getting requested a lot? If not I guess that could make it easier?

Reply:
@DFurnes ended up just going for it and trying this out. Ultimately it'll make adding more legacy redirects easier in the future! Lemme know if all looks good and works in a dry-run 😉